pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc-view-results.adoc

```diff
@@ -0,0 +1,11 @@
+[id='view-results_{context}']
+
+.Viewing search results
+The search results are presented in a table organized into columns with the following headings:
+
+* `Title` - the module title
+* `Published` - the publication status, `Published` or `Not published`
+* `Draft uploaded` - the date the draft was uploaded
+* `Module Type` - the module type, `Concept`, `Procedure`, or `Reference`
```

Comment:
Similar to the filters in another file, only these four are actual column headings. The next two items in this list describe an actual procedure.